
<div style="margin-left: 10px">
<h1>Universidade Federal de Mato Grosso do Sul</h1>
Campus de Campo Grande <br>
Estatística – Prof. Cássio Pinho dos Reis
</div>
</div>


<left><h3>8ª LISTA DE EXERCÍCIOS - parte 4</h3></left> <br>
<b>Turma:</b> Engenharia de Software <br>
<b>RGA:</b> 2021.1906.069-7<br>
<b>Aluno:</b> Maycon Felipe da Silva Mota



---





In [22]:
import math
import statistics as stats
from scipy.stats import norm


In [23]:
# Definir as funções que irei usar nos exercicios abaixos

In [24]:
def calcularSignificancia(valor):
    calculo = round(1-valor, 2) 
    print(f"Pesquisar valor Z: {(50 -((calculo*100))/2)/100}")
    return calculo 

def calcularLimInf(proporcao, tamanhoAmostral, valorZ):
    calculo = proporcao - valorZ * math.sqrt((proporcao*(1-proporcao))/tamanhoAmostral)
    return calculo

def calcularLimSup(proporcao, tamanhoAmostral, valorZ):
    calculo = proporcao + valorZ * math.sqrt((proporcao*(1-proporcao))/tamanhoAmostral)
    return calculo

proporcao =  93/100
valor_Z = calcularSignificancia(0.95) # 1.96
limite_inf = calcularLimInf(proporcao, 100, 1.96)
limite_sup = calcularLimSup(proporcao, 100, 1.96)
intervalo_confiança = 0.95

print(f"Resultado ======> \n P( {limite_inf:.3f} =< u =< {limite_sup:.3f} = {intervalo_confiança:.2%} )")

Pesquisar valor Z: 0.475
Resultado ======> 
 P( 0.880 =< u =< 0.980 = 95.00% )


# 1 - Uma amostra aleatória de 60 unidades de certo produto químico apresentou preço médio, por kg, de R\$ 45,21. Determine um intervalo de confiança de 90% para estimar a média populacional do preço desse produto, sabendo que o desvio padrão populacional é de R$ 6,32

In [25]:
int_conf_90 = norm.interval(0.90, loc=45.21, scale=6.32 / math.sqrt(60)) 
print("intervalo de confiança 90.0%: {:.3f} a {:.3f}".format(int_conf_90[0], int_conf_90[1]))


intervalo de confiança 90.0%: 43.868 a 46.552


<p> O intervalo [43.87% ; 46.55%] contém a verdadeira proporção de peças defeituosas com 90% de confiança </p>

# 2ª Questão – Uma amostra aleatória de 40 latas de cerveja de uma indústria de bebidas revelou uma quantidade média de 347ml de cerveja inserida nas latas. Admitindo-se que o desvio padrão das quantidades de cerveja inserido é de 4ml, construa um intervalo de confiança de 95% para a verdadeira média de cerveja nas latas

In [26]:
int_conf_95 = norm.interval(0.95, loc=347, scale=4 / math.sqrt(40)) 
print("intervalo de confiança 95.0%: {:.3f} a {:.3f}".format(int_conf_95[0], int_conf_95[1]))


intervalo de confiança 95.0%: 345.760 a 348.240


<p> O intervalo [345.760 ; 348.240] contém a verdadeira proporção de cervejas nas latas </p>

# 3ª Questão – Uma engenheira selecionou uma amostra aleatória de 12 medidas da tensão de ruptura de certo tipo de fio de algodão, a qual apresentou uma tensão média de ruptura de 7,38kg e um desvio padrão de 1,24 kg. Determine um intervalo de 95% de confiança para a verdadeira média de ruptura desse tipo de fio.

In [27]:
int_conf_95 = norm.interval(0.95, loc=7.38, scale=1.24 / math.sqrt(12)) 
print("intervalo de confiança 95.0%: {:.3f} a {:.3f}".format(int_conf_95[0], int_conf_95[1]))


intervalo de confiança 95.0%: 6.678 a 8.082


<p> O intervalo [6.678 ; 8.082] contém a verdadeira média de ruptura dos fios com intervalo de confiança de 95% é de  6.678 a 8.082. </p>